dataland-frontend/tests/e2e/fixtures/common/DataPointFixtures.ts

```diff
@@ -1,5 +1,10 @@
 import { faker } from "@faker-js/faker";
-import { CompanyReportReference, DataPointBigDecimal, DataPointYesNo, QualityOptions } from "@clients/backend";
+import {
```

Comment:
The second line of "generateReferencedReports()" is not necessary, if you force `arrayElements()` to pick minimum of one element.
See this code snippet to see how you can do that:

![image](https://github.com/d-fine/Dataland/assets/99255296/2039b59d-854c-4434-bf5f-95e6faf403e8)

Reply:
Changed it, please check if this is what you meant

Reply:
Changed the lenght, the variable name and refreshed the fake fixtures

Reply:
Thx 👍